tests/ci/cloudbuild.yaml

```diff
@@ -7,8 +7,8 @@ steps:
   - |
     git clone \
       --depth=1 \
-      --branch=release \
-      https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools
+      --branch=add-ducktape-task \
```

Comment:
?

Reply:
that's temporary since those changes on vtools cannot ve berged to `release` cause they'll break other builds

Reply:
ack